test/integration_test/operator_test.go

```diff
@@ -0,0 +1,125 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/pkg/asyncdr"
+	"github.com/portworx/torpedo/pkg/log"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	kubeconfigDirectory   = "/tmp"
+	backupLocation        = "s3"
+	backupSecret          = "s3secret"
+	clusterPairName       = "bid-pair"
+	migNamePref           = "operator-"
+	includeVolumesFlag    = true
+	includeResourcesFlag  = true
+	startApplicationsFlag = true
+	appNameMongo          = "mongo"
+	appPathMongo          = "/stork-specs/mongocr.yaml"
+	appNameKafka          = "kafka"
+	appPathKafka          = "/stork-specs/kafkacr.yaml"
+)
+
+func TestOperatorMig(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+	logrus.Infof("Using stork volume driver: %s", volumeDriverName)
+	t.Run("testMongoMig", testMongoMig)
+	t.Run("testKafkaMig", testKafkaMig)
+}
+
+func testMongoMig(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+	validateAndDestroyCrMigration(t, appNameMongo, appPathMongo)
+}
+
+func testKafkaMig(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+	validateAndDestroyCrMigration(t, appNameKafka, appPathKafka)
+}
+
+func validateAndDestroyCrMigration(t *testing.T, appName string, appPath string) {
+	err := setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
```

Comment:
Add new line after every require that way the code is more readable IMO.

Reply:
Done.